src/snapshot.c

```diff
@@ -707,6 +709,11 @@ static void clientSessionRDBSave(RedisModuleIO *rdb)
     ClientSession *client_session;
     while (RedisModule_DictNextC(iter, NULL, (void **) &client_session) != NULL) {
         RedisModule_SaveUnsigned(rdb, client_session->client_id);
+        if (RedisModule_GetClientFlags(client_session->client)) {
```

Comment:
see my comment for `isClientSessionDirty()`

Reply:
done